service/src/test/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityUserServiceTest.java

```diff
@@ -117,8 +121,14 @@ void shouldAddCallerUserIdToACaseWhenServiceThrowsAnException()  {
 
     @Test
     void shouldNotThrowExceptionWhenCallToUpdateCaseRoleEndpointFailsForOneUser() {
-        willThrow(new RetryableException(500, "Some error", null, null)).given(caseUserApi).updateCaseRolesForUser(
-            eq(AUTH_TOKEN), eq(SERVICE_AUTH_TOKEN), eq(CASE_ID), eq("1"), refEq(new CaseUser("1", caseRoles)));
+
+        willThrow(new RetryableException(500,
+            "Some error",
+            Request.HttpMethod.GET,
```

Comment:
Static import on at least Request, feel like that part is a bit unnecessary

Reply:
done